MinPlatformPkg/Test/Library/TestPointCheckLib/DxeTestPointCheckLib.inf

```diff
@@ -111,3 +111,6 @@
 [Pcd]
   gMinPlatformPkgTokenSpaceGuid.PcdTestPointIbvPlatformFeature
   gMinPlatformPkgTokenSpaceGuid.PcdTestPointIbvPlatformExemptPciBme # MU_CHANGE - TCBZ3541
+
+[Depex]
+  gEfiSmmAccess2ProtocolGuid
```

Comment:
Can you leave a brief comment for what this DEPEX is for? It will help in the future when considering loading order changes.

Reply:
Removed the DEPEX.  It was part of a change I was testing and later removed.